### PR TITLE
feat: adds portuguese language

### PR DESCRIPTION
**Pull request by prosazhin**
This pull request adds the Portuguese version of the pages.

Pages on which the translation should be:
- [x] header
- [x] footer
- [x] SEO
- [x] [home](https://deploy-preview-493--ebpfio.netlify.app/pt-br/)
- [x] [what-is-ebpf](https://deploy-preview-493--ebpfio.netlify.app/pt-br/what-is-ebpf/)
- [x] [get-started](https://deploy-preview-493--ebpfio.netlify.app/pt-br/get-started/)
- [x] [case-studies](https://deploy-preview-493--ebpfio.netlify.app/pt-br/case-studies/)
- [x] [labs](https://deploy-preview-493--ebpfio.netlify.app/pt-br/labs/)
- [x] [events](https://deploy-preview-493--ebpfio.netlify.app/pt-br/events/)
- [x] [applications](https://deploy-preview-493--ebpfio.netlify.app/pt-br/applications/)
- [x] [infrastructure](https://deploy-preview-493--ebpfio.netlify.app/pt-br/infrastructure/)
- [x] [contribute](https://deploy-preview-493--ebpfio.netlify.app/pt-br/contribute/)

### CODE REVIEW

**Comment by prosazhin** [comment]
@xmulligan Hello. I don't have enough translations to complete the localization for Portuguese. I have made a list to complete this task.

Pages on which the translation should be:
- [ ] header
- [ ] footer
- [x] [home](https://deploy-preview-493--ebpfio.netlify.app/pt/)
- [ ] [what-is-ebpf](https://deploy-preview-493--ebpfio.netlify.app/pt/what-is-ebpf/)
- [ ] [get-started](https://deploy-preview-493--ebpfio.netlify.app/pt/get-started/)
- [ ] [case-studies](https://deploy-preview-493--ebpfio.netlify.app/pt/case-studies/)
- [ ] [labs](https://deploy-preview-493--ebpfio.netlify.app/pt/labs/)
- [x] [events](https://deploy-preview-493--ebpfio.netlify.app/pt/events/)
- [ ] [applications](https://deploy-preview-493--ebpfio.netlify.app/pt/applications/)
- [ ] [infrastructure](https://deploy-preview-493--ebpfio.netlify.app/pt/infrastructure/)
- [ ] [contribute](https://deploy-preview-493--ebpfio.netlify.app/pt/contribute/)

**Comment by xmulligan** [comment]
Which parts are missing?

**Comment by prosazhin** [comment]
@xmulligan I didn't find a translation for two paragraphs and a title on the [what-is-ebpf](https://deploy-preview-493--ebpfio.netlify.app/pt/what-is-ebpf/) page.
Снимок экрана 2023-10-04 в 12 49 15
Снимок экрана 2023-10-04 в 12 49 22

**Comment by prosazhin** [comment]
@xmulligan Also, I didn’t find translations for the headings and buttons for the [get-started](https://deploy-preview-493--ebpfio.netlify.app/pt/get-started/) page.
Снимок экрана 2023-10-04 в 12 52 25
Снимок экрана 2023-10-04 в 12 52 45

**Comment by prosazhin** [comment]
@xmulligan I'm sorry. For the get-started page, the translation is available in full. I made all the translations that were available and there is not enough translation for the [contribute](https://deploy-preview-493--ebpfio.netlify.app/pt/contribute/) page.

**Comment by ebpfbrasil** [comment]
@prosazhin @xmulligan (portuguese version)


Se um gancho predeterminado não existe para uma necessidade em particular , É possível criar um kernel probe (kprobe)ou usuário probe (uprobe) para anexar programas eBPF praticamente em qualquer lugar do kernel ou aplicações do usuário.


Como os programas de eBPF são escritos ?

Em muitos cenários,  eBPF não é usado diretamente mas sim indiretamente através de projetos como Cilium, bcc ou bpftrace que fornecem uma abstração em cima do eBPF e não necessitam de um programa de escrita diretamente, mas ao invés disso Oferecem a habilidade de especificar definições baseadas nas intenções que são implementadas com eBPF.

**Comment by Hudsoncoutinho** [comment]
@prosazhin @xmulligan (portuguese version)

Se um gancho predeterminado não existe para uma necessidade em particular , É possível criar um kernel probe (kprobe)ou usuário probe (uprobe) para anexar programas eBPF praticamente em qualquer lugar do kernel ou aplicações do usuário.

Como os programas de eBPF são escritos ?

Em muitos cenários, eBPF não é usado diretamente mas sim indiretamente através de projetos como Cilium, bcc ou bpftrace que fornecem uma abstração em cima do eBPF e não necessitam de um programa de escrita diretamente, mas ao invés disso oferecem a habilidade de especificar definições baseadas nas intenções que são implementadas com eBPF.

**Comment by prosazhin** [comment]
@xmulligan All that's left is the translation for the contribute page.

**Comment by xmulligan** [comment]
Thanks @Hudsoncoutinho can you help with that too?

File: config/languages.js
Comment (cpanato) [inline config/languages.js:14]: I think here should be 
```suggestion
      name: 'Português',
```

File: content/static-pages/what-is-ebpf/index.fr-fr.md
Comment (cpanato) [inline content/static-pages/what-is-ebpf/index.fr-fr.md:162]: Is this on purpose?

**Review by cpanato — COMMENTED** [review]
before continue, need to know if this translate will be `pt` or `pt-br` 
(pt=European portuguese; pt-br=Brazilian portuguese)

thanks

**Comment by marcsvll** [comment]
> before continue, need to know if this translate will be `pt` or `pt-br` (pt=European portuguese; pt-br=Brazilian portuguese)
> 
> thanks

I had the same question, but based on the [icon](https://github.com/ebpf-io/ebpf.io-website/blob/36c04a1fb8baaf08d18ddd7b43ff3d95282ac499/src/icons/languages/pt.inline.svg) chosen to represent the language, I'm assuming it is Portuguese from Portugal.

File: content/static-pages/what-is-ebpf/index.pt.md
Comment (marcsvll) [inline content/static-pages/what-is-ebpf/index.pt.md:10]: The original text has the following sentence: ".. that can run **sandboxed** programs in a privileged context..."

The translation has no references to the term *sandboxed*, so the idea behind the context is lost. But this word does not have a direct translation into Portuguese. Commonly, when there is no corresponding term in Portuguese, we either leave the term in the source language and use italic formatting (example: "que pode executar programas em _sandbox_ em um contexto privilegiado" ) or look for a word that can carry a similar meaning. I would like to know the review preference in this case.

**Comment by xmulligan** [comment]
This one should be Brazilian Portuguese. We are also working on a Portugal one, but that is separate.

@prosazhin can you update the flag to Brazil?

**Comment by Hudsoncoutinho** [comment]
@xmulligan 
what needs to be translated into Portuguese? please tell me the excerpt so I can help with that

File: content/static-pages/what-is-ebpf/index.pt.md
Comment (xmulligan) [reply content/static-pages/what-is-ebpf/index.pt.md:10]: I would probably go with the former so _sandbox_

**Comment by xmulligan** [comment]
@Hudsoncoutinho for the contribute page we need

How to Contribute to eBPF
eBPF consists of many communities including the eBPF runtime in the Linux kernel, various development SDKs and libraries, a series of large projects using eBPF, and a wide set of smaller projects and tools. There are therefore many ways to contribute to the eBPF community and get involved. The following list of ideas can serve as a starting point on how you can get involved and contribute to the community.


Maintainers of the eBPF runtime in the Linux kernel

Contribute to the eBPF runtime in the Linux kernel
The Linux kernel community is maintaining separate Git trees for the eBPF subsystem to manage all developer contributions. The trees are maintained by:

Alexei Starovoitov, Facebook
Daniel Borkmann, Cilium/Isovalent
Working on the eBPF runtime can be incredibly rewarding as it builds the foundation for software being written that runs on millions of systems and devices. Work includes a wide range of topics from adding support to additional hooks, adding new program types, improvements to the eBPF verifier, adding JIT support for additional CPU architectures, extending the set of BPF helpers, adding new test cases to BPF’s test infrastructure, or improvements to bpftool and libbpf.


Contribute to an eBPF project
The list of eBPF-based projects is long and growing. It will be simple to find a project that sparks interest. Check out the eBPF landscape to see an overview of eBPF-based projects.

Many of the listed eBPF-based projects maintain a list of good-first-issue-labeled tasks which are scoped to not require extensive project specific knowledge and provide a great opportunity to get hands-on quickly.

Contribute to the eBPF development toolchains
Even though the Linux kernel provides minimal libraries and APIs to use eBPF directly, the majority of projects leverage an eBPF development toolchain to simplify use of eBPF and share common code among other projects.

Check out eBPF development toolchains for a frequently updated list of projects in this space. You will find information on how to get involved on each project website directly.

EBPF DEVELOPMENT TOOLCHAINS

Contribute to ebpf.io
Do you have a writing talent? Are you good with diagrams? The content of the ebpf.io website is licensed under a Creative Commons Attribution 4.0 International License. If you want to get involved, visit the GitHub repository. Make sure to also join the #ebpf-website Slack channel to get in touch with the team working on the documentation. Share what you have learned by improving the documentation or write additional tutorials.

File: content/static-pages/what-is-ebpf/index.pt.md
Comment (marcsvll) [reply content/static-pages/what-is-ebpf/index.pt.md:10]:
```suggestion
O eBPF é uma tecnologia revolucionária com origens no kernel Linux que pode executar programas em _sandbox_ em um contexto privilegiado, como o kernel do sistema operacional. Ele é usado para estender com segurança e eficiência as capacidades do kernel sem a necessidade de alterar o código-fonte do kernel ou carregar módulos do kernel.
```

File: src/data/pages/contribute/index.js
Comment (marcsvll) [inline src/data/pages/contribute/index.js:258]:
```suggestion
  pt: {
    hero: {
      title: 'Como Contribuir com o eBPF',
      description:
        'O eBPF consiste em muitas comunidades, incluindo o _runtime_ do eBPF no kernel Linux, vários SDKs e bibliotecas de desenvolvimento, uma série de grandes projetos usando eBPF e um amplo conjunto de projetos e ferramentas menores. Existem, portanto, muitas maneiras de contribuir com a comunidade eBPF e de se envolver. A lista de ideias a seguir pode servir como ponto de partida sobre como você pode se envolver e contribuir com a comunidade.',
      card: {
        title: 'Contribua com o runtime do eBPF no kernel Linux',
        description:
          'A comunidade do kernel Linux mantém árvores Git separadas para o subsistema eBPF para gerenciar todas as contribuições dos desenvolvedores. As árvores são mantidas por:',
        secondDescription:
          'Trabalhar no tempo de execução do eBPF pode ser incrivelmente gratificante, pois constrói a base para a gravação de software que roda em milhões de sistemas e dispositivos. O trabalho inclui uma ampla gama de tópicos, desde adição de suporte a ganchos adicionais, adição de novos tipos de programas, melhorias no verificador eBPF, adição de suporte JIT para arquiteturas de CPU adicionais, extensão do conjunto de auxiliares BPF, adição de novos casos de teste à infraestrutura de teste do BPF, ou melhorias no bpftool e libbpf.',
        authors: [
          {
            name: 'Alexei Starovoitov',
            company: 'Facebook',
          },
          {
            name: 'Daniel Borkmann',
            company: 'Cilium/Isovalent',
          },
        ],
        links: [
          {
            text: 'Git trees',
            to: 'https://git.kernel.org/?q=BPF+Group',
          },
          {
            text: 'Linux kernel eBPF runtime',
            to: '/infrastructure/#linux-kernel',
          },
        ],
      },
    },
    howToContribute: {
      items: [
        {
          icon: 'file',
          title: 'Contribua para um projeto eBPF',
          description:
            '<p>A lista de projetos baseados em eBPF é longa e crescente. Será simples encontrar um projeto que desperte interesse. Confira o panorama do eBPF para ter uma visão geral dos projetos baseados em eBPF.</p><p>Muitos dos projetos baseados em eBPF listados mantêm uma lista de boas tarefas rotuladas como o primeiro problema, com escopo que não requer amplo conhecimento específico do projeto e oferece uma grande oportunidade de colocar a mão na massa rapidamente.</p>',
          links: [
            {
              text: 'eBPF landscape',
              to: '/applications',
            },
          ],
        },
        {
          icon: 'gear',
          title: 'Contribua com as ferramentas de desenvolvimento do eBPF',
          description:
            '<p>Embora o kernel Linux forneça bibliotecas e APIs mínimas para usar o eBPF diretamente, a maioria dos projetos utiliza uma cadeia de ferramentas de desenvolvimento de eBPF para simplificar o uso do eBPF e compartilhar código comum entre outros projetos.</p><p>Confira as cadeias de ferramentas de desenvolvimento eBPF para obter uma lista atualizada frequentemente de projetos neste espaço. Você encontrará informações sobre como se envolver diretamente no site de cada projeto.</p>',
          links: [
            {
              text: 'eBPF development toolchains',
              to: '/what-is-ebpf/#development-toolchains',
            },
          ],
        },
        {
          icon: 'pen',
          title: 'Contribua com ebpf.io',
          description:
            '<p>Você tem talento para escrever? Você é bom com diagramas?O conteúdo do site ebpf.io está licenciado sob uma licença _Creative Commons Attribution 4.0 International_. Se você quiser se envolver, visite o repositório no GitHub. Não deixe de entrar também no canal #ebpf-website no Slack para entrar em contato com a equipe que está trabalhando na documentação. Compartilhe o que você aprendeu melhorando a documentação ou escrevendo tutoriais adicionais.</p>',
          links: [
            {
              text: 'License',
              to: 'https://creativecommons.org/licenses/by/4.0/',
              target: '_blank',
            },
            {
              text: 'GitHub repository',
              to: 'https://github.com/ebpf-io/ebpf.io-website',
              target: '_blank',
            },
            {
              text: 'Slack',
              to: 'https://ebpf.io/slack',
            },
          ],
        },
      ],
    },
  },
```

Comment (marcsvll) [reply src/data/pages/contribute/index.js:258]: @prosazhin @xmulligan I made the suggestion with the translation into Brazilian Portuguese for the contribute page.

File: src/data/pages/get-started/index.js
Comment (marcsvll) [inline src/data/pages/get-started/index.js:282]:
```suggestion
            'Tomando emprestado o exemplo do opensnoop do relatório de Liz Rice, este laboratório ensina como manipular uma ferramenta eBPF, observá-la carregando seus componentes e até mesmo adicionar seu próprio rastreamento ao código-fonte eBPF.',
```

File: src/data/pages/home/index.js
Comment (marcsvll) [inline src/data/pages/home/index.js:277]:
```suggestion
      blackButtonTitle: 'Cenário do Projeto',
```

Comment (marcsvll) [reply src/data/pages/home/index.js:277]: This case also needs a bit more complex and direct translation. I adopted the word "Scenario" because "Landscape" in Portuguese is commonly only used for natural images or architectural context.

File: src/data/pages/contribute/index.js
Comment (xmulligan) [reply src/data/pages/contribute/index.js:258]: Great, @Hudsoncoutinho what do you think?

**Comment by prosazhin** [comment]
@xmulligan Hello. Please do not merge. I will make all the necessary changes.

**Comment by prosazhin** [comment]
@xmulligan I have made all the necessary changes, please check

File: src/data/pages/contribute/index.js
Comment (marcsvll) [inline src/data/pages/contribute/index.js:179]:
```suggestion
        'O eBPF consiste em muitas comunidades, incluindo o runtime do eBPF no kernel Linux, vários SDKs e bibliotecas de desenvolvimento, uma série de grandes projetos usando eBPF e um amplo conjunto de projetos e ferramentas menores. Existem, portanto, muitas maneiras de contribuir com a comunidade eBPF e de se envolver. A lista de ideias a seguir pode servir como ponto de partida sobre como você pode se envolver e contribuir com a comunidade.',
```

Comment (marcsvll) [reply src/data/pages/contribute/index.js:179]: I used Markdown formatting when I made this suggestion but it doesn't match the file format.

**Review by marcsvll — APPROVED** [review]

**Comment by prosazhin** [comment]
@xmulligan Hello. I have made all the recent changes from the comments. It can be reviewed now.

**Review by americano98 — APPROVED** [review]

**Comment by xmulligan** [comment]
Thanks for all the help @marcsvll @Hudsoncoutinho @cpanato ! Any last comments or changes before we merge the page?

**Comment by marcsvll** [comment]
> Thanks for all the help @marcsvll @Hudsoncoutinho @cpanato ! Any last comments or changes before we merge the page?

Hello @xmulligan, apologies for the delay in my response. I have completed the review, and everything looks good to me.

**Review by xmulligan — APPROVED** [review]

**Comment by cpanato** [comment]
i was a bit late for the final review :( but i've read and that makes sense to me, thanks for doing that

**Pull request merged**
